### DISCUSSION TITLE
Split up long alert messages

### DISCUSSION THREAD

**Discussion by dgrisham** [grafana/grafana#46208] [Alerting]
Hi! I'm using a custom alerting template to send alerts to a Discord channel. When testing the alerts I hit a case where, I believe, the message was too long to send to Discord because there were two many alerts fired at once. I noticed a `400 Bad Request` in the Grafana server logs for the Discord alert post. I suspected it was the length, so I triggered the first and second half of the failed set of alerts separately (to ensure one of the halves wasn't formatted in some way that was causing Discord to reject the request), and each one succeeded. Then I compared the length of each half to the total, and the total length was indeed longer than the maximum discord message length.

So, my question is: What would be the best way to handle this? I didn't see any issues where this has come up, but perhaps I missed it. Is there a way to manage this through Grafana right now, or would this be a new feature?

My server is running Ubuntu 20.04, with Grafana version:
```
$ grafana-server -v
Version 8.4.1 (commit: 53f5c6a44c, branch: HEAD)
```

**Comment by hunkjun** [comment]
i face the same problem. i will use template to split the message.

**Reply by WiFiAdapter** [reply]
Hey, I have the same problem. Can you elaborate on how to split the message using a template?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
